src/evrythng/tests/test_entities.py

```diff
@@ -0,0 +1,122 @@
+import unittest
+
+from evrythng.entities import actions
+from evrythng.entities import products
+
+
+class TestProductMethods(unittest.TestCase):
+
+    '''testing the following methods:
+
+    create_product +
+    list_products
+    read_product +
+    update_product
+    delete_product
+
+    on a test EVT account by Dmitry Shulgin.
+
+    '''
+
+    def test1_creation(self):
```

Comment:
Ohh, i see. 
http://stackoverflow.com/questions/6854658/explain-the-setup-and-teardown-methods-used-in-test-cases